src/json.cpp

```diff
@@ -117,7 +117,7 @@ void JsonObject::report_unvisited() const
             const std::string &name = p.first;
             if( !visited_members.count( name ) && !string_starts_with( name, "//" ) ) {
                 try {
-                    throw_error( string_format( "Failed to visit member %s in JsonObject", name ), name );
+                    throw_error( string_format( "Invalid field name \"%s\" in JSON data", name ), name );
```

Comment:
This can also occur when the field is placed in the wrong place.
```suggestion
                    throw_error( string_format( "Invalid or misplaced field name \"%s\" in JSON data", name ), name );
```